src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -412,7 +412,9 @@ export const TradeConfirm = () => {
     [fees?.tradeFeeSource],
   )
 
-  const shouldShowDonationOption = isTHORChainSwap
+  const is0xSwap = useMemo(() => fees?.tradeFeeSource === SwapperName.Zrx, [fees?.tradeFeeSource])
```

Comment:
Since we're doing the same with `isTHORChainSwap`, we might want to abstract these into a set of pure helpers, to make the view-layer slightly cleaner. Not a big deal obviously but I feel like that's a pattern we might end up needing quite a bunch

Reply:
Added a partial abstraction in https://github.com/shapeshift/web/pull/4392/commits/4dd487bf2027cf45727de14a23371ec02e451d05